blog: Welcome Ratify Joins the Notary Project (#440)
notaryproject/notaryproject.dev: content/en/blog/2025/welcome-ratify-joined-notaryproject.md

---
title: Ratify Joins the Notary Project - Strengthening Software Supply Chain Security Together!
author:  "Notary Project Community"
date: 2025-06-02
draft: false
---

![notary project welcomes ratify](/notation-ratify.svg)

We’re excited to announce that [Ratify](https://ratify.dev/) has officially joined the Notary Project as a subproject after the [vote](https://github.com/notaryproject/.github/issues/81) passed in the community! 🎉 This marks a significant step forward in our shared mission to deliver **secure, transparent, and trusted** software supply chain for the cloud-native ecosystem.

> The Notary Project is building a set of specifications and reference implementations to secure the integrity of container images and other OCI artifacts. With Ratify’s addition, we expand our surface to policy-based verification and extensibility—helping organizations validate signatures, SBOM, vulnerability scanning report, and other security metadata of container images before deploying to Kubernetes.

Author: Added CI/CD scenario.

By welcoming Ratify as an official subproject, the Notary Project now offers a broader and more opinionated solution stack for securing software supply chain:

- Notation enables signing of OCI artifacts in CI/CD pipelines.

> - Ratify enforces signature and other supply chain metadata verification policies by working with Gatekeeper in runtime, Kubernetes, and CI/CD.

Reviewer: ```suggestion
- Ratify enforces signature and other supply chain metadata verification policies by working with Gatekeeper on Kubernetes, and within CI/CD pipelines.
```

Author: @yizha1 ACK. I removed "Gatekeeper" and updated wording to make it clearer and simpler.
